app/assets/javascripts/dashboard/dashboard.js

```diff
@@ -328,8 +329,17 @@ MinionPoller = {
   },
 
   alertFailedBootstrap: function() {
-    if (!$('.failed-bootstrap-alert').length) {
-      showAlert('At least one of the nodes is in a failed state. Please run "supportconfig" on the failed node(s) to gather the logs.', 'alert', 'failed-bootstrap-alert');
+    var cachedFailedLastOrchestration = window.localStorage.getItem('failedLastOrchestrationAt');
+
+    if (!$('.failed-bootstrap-alert').length &&
+        cachedFailedLastOrchestration !== State.lastOrchestrationAt) {
```

Comment:
would it make sense to do an early return here instead of wrapping the code in a conditional block? not sure about js guidelines

Reply:
I personally don't have a strong opinion about this. I'd say that if the condition were simple, it's fine to do the early return. In this case we have a double condition and we can't write it as in ruby (return if ...). So I guess it's fine to leave it as it is.

Reply:
I agree with @MaximilianMeister; I'd prefer to see a guard function here. If Rubocop were able, he'd complain :P